### PR TITLE
React issue1

### PR DESCRIPTION
**Pull request by ai-xiaihai**

### CODE REVIEW

**Comment by ai-xiaihai** [comment]
[React Issue 1: Add Footer](https://trello.com/c/XfVGonMc/188-react-issue-1-add-footer)

File: app/javascript/image_sharing/components/FeedbackForm.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Form, FormGroup, Label, Input, Button } from 'reactstrap';
+
+export default class FeedbackForm extends React.Component {
+  render() {
+    return (
+      <Form>
+        <FormGroup>
+          <Label for="commentSource">Your name:</Label>
+          <Input type="text" id="commentSource" />
```
Comment (eldonchen0501) [inline app/javascript/image_sharing/components/FeedbackForm.js]: the type of `Input` is `text` by default so we don't need to specify it here.

File: app/javascript/image_sharing/components/FeedbackForm.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        </FormGroup>
+        <FormGroup>
+          <Label for="commentContent">Comment:</Label>
+          <Input type="text" id="commentContent" />
```
Comment (eldonchen0501) [inline app/javascript/image_sharing/components/FeedbackForm.js]: According to the mockup, the type of this input should be `textarea`

File: app/javascript/image_sharing/components/FeedbackForm.js (hunk 1 above)
Comment (ai-xiaihai) [reply app/javascript/image_sharing/components/FeedbackForm.js]: I modified this.

File: app/javascript/image_sharing/components/FeedbackForm.js (hunk 2 above)
Comment (ai-xiaihai) [reply app/javascript/image_sharing/components/FeedbackForm.js]: I modified this.

**Pull request merged**